tests/cases/conformance/es6/modules/multipleDefaultExports03.ts

```diff
@@ -0,0 +1,10 @@
+// @module: commonjs
+// @target: ES5
+
+// @filename: m1.ts
+export default class C {
+}
+
+// @filename: m2.ts
```

Comment:
Sorry, I didn't want two separate files, I wanted both declarations to occur in the same file.

Also, can you add multipleDefaultExports04?

```TypeScript
export default function f() {
}
export default function f() {
}
```